app/javascript/app/utils/utils.js

```diff
@@ -35,6 +35,11 @@ export function compareIndexByKey(attribute) {
   };
 }
 
+const r2lWrittedCountries = ['IRQ'];
```

Comment:
Couldn't find the Iraq full NDC text. The code seems alright. Check http://www.i18nguy.com/temp/rtl.html
To find the whole rtl countries list:

Country/Region | Script | Direction1 | Language
-- | -- | -- | --
Afghanistan | Arabic | RTL | Pashto
Israel | Hebrew | RTL | Hebrew
Middle East | Arabic | RTL | Arabic
Pakistan | Urdu | RTL | Urdu